extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -24,26 +100,120 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
         <div
           class={
             'i-amphtml-amp-story-audio-sticker-container' +
-            (this.element.getAttribute('size') === 'small'
-              ? ' small'
-              : ' large')
+            (this.element.getAttribute('size') === 'large'
+              ? ' large'
+              : ' small')
           }
         >
           {this.element}
         </div>
       </div>
     );
+
+    const pageEl = closestAncestorElementBySelector(
+      this.element,
+      'amp-story-page'
+    );
+    const pageId = pageEl.getAttribute('id');
+
+    Services.storyStoreServiceForOrNull(this.win).then((storeService) => {
+      this.storeService_ = storeService;
+      this.initializeListeners_();
+      this.storeService_.subscribe(
+        StateProperty.CURRENT_PAGE_ID,
+        (currentPageId) =>
+          this.toggleStickerState_('on-active-page', pageId === currentPageId)
+      );
+    });
+  }
+
+  /** @private */
+  maybeInitializeDefaultSticker_() {
+    this.defaultSticker_ = this.getDefaultSticker_();
+    if (!this.defaultSticker_) {
+      return;
+    }
+
+    // Remove all existing child elements and add only pretap and posttap elements.
+    this.element.replaceChildren();
+    this.element.appendChild(
+      <amp-story-audio-sticker-pretap>
+        <amp-img
+          width={DEFAULT_STICKERS[this.defaultSticker_].width}
+          height={DEFAULT_STICKERS[this.defaultSticker_].height}
+          layout="responsive"
+        ></amp-img>
+      </amp-story-audio-sticker-pretap>
+    );
+    this.element.appendChild(
+      <amp-story-audio-sticker-posttap>
+        <amp-img
+          width={DEFAULT_STICKERS[this.defaultSticker_].width}
+          height={DEFAULT_STICKERS[this.defaultSticker_].height}
+          layout="responsive"
+        ></amp-img>
+      </amp-story-audio-sticker-posttap>
+    );
+  }
+
+  /**
+   * Return the specified default sticker or null if there's custom sticker provided.
+   * Return fallback sticker is both default and custom sticker config are not specified at all.
+   * @private
+   * @return {string}
+   */
+  getDefaultSticker_() {
+    if (
+      scopedQuerySelector(this.element, STICKER_PRETAP_TAG) &&
+      scopedQuerySelector(this.element, STICKER_POSTTAP_TAG)
+    ) {
+      return null;
+    }
+
+    const stickerAttr = this.element.getAttribute('sticker');
+    return stickerAttr || FALLBACK_DEFAULT_STICKER;
```

Comment:
Check if stickerAttr is in the map, otherwise return the fallback since it's an invalid sticker.

Reply:
Done.

Yeah I thought we should always rely on validator for this kind of check. Do we intentionally do this kind of checks in both js code and validator in general?